### PR TITLE
Instant transfer

### PR DESCRIPTION
**Pull request by lera**

### CODE REVIEW

File: src/routes/instantCreditTransfer.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+export const getInstantReachability = (req: Request, res: Response) => {
+  const { iban } = req.params;
+
+  log.info(`Checking instant reachability for iban ${iban}`);
+
+  let result = true;
+
+  if (IBAN.isValid(iban)) {
+    result = false;
```
Comment (p-janik) [inline src/routes/instantCreditTransfer.ts:29]: hmm shouldn't this be true?

Comment (lera) [reply src/routes/instantCreditTransfer.ts:29]: ooops! wrong condition

Comment (lera) [reply src/routes/instantCreditTransfer.ts:29]: changed to not valid iban

**Review by p-janik — APPROVED** [review]
👍

**Pull request merged**